resources/views/admin/checkout.blade.php

```diff
@@ -13,8 +13,153 @@
         <div class="card">
             <div class="card-content">
                 <span class="card-title">@lang('checkout.checkout')</span>
+                <blockquote>
+                    @lang('checkout.current_balance'): 
+                    <b class="coli-text text-orange"> {{ number_format($current_balance, 0, '.', ' ') }} Ft</b>.<br>
+                    @lang('checkout.current_balance_in_checkout'): 
+                    <b class="coli-text text-orange"> {{ number_format($current_balance_in_checkout, 0, '.', ' ') }} Ft</b>.<br>
+                </blockquote>
+                @if(Auth::user()->isSysAdmin())
```

Comment:
You should create a policy and use `@can('addPayment', \App\Checkout::admin())`.
See app\Policies\CheckoutPolicy.php